pkg/model/keycloak_deployment.go

```diff
@@ -20,6 +20,7 @@ func KeycloakDeployment(cr *v1alpha1.Keycloak) *v13.StatefulSet {
 			},
 		},
 		Spec: v13.StatefulSetSpec{
+			Replicas: &[]int32{int32(cr.Spec.Instances)}[0],
```

Comment:
I just realized this is wrong... What if there's no instances??!!

Reply:
Actually we don't use defaults in our CR (we probably should). In this particular case, I would like to double check if everything is fine. I'll implement an additional check.